services/src/handlers/datasets.rs

```diff
@@ -282,7 +287,10 @@ fn auto_detect_meta_data_definition(main_file_path: &Path) -> Result<MetaDataDef
         result_descriptor: VectorResultDescriptor {
             data_type: vector_type,
             spatial_reference: spatial_reference.into(),
-            columns: columns_map,
+            columns: columns_map
+                .into_iter()
+                .filter_map(|(k, v)| v.try_into().map(|v| (k, v)).ok())
```

Comment:
maybe add a comment that you ignore all columns here that don't have a corresponding GE type

Reply:
c5ffd6ec608bf584f64d7f98f94be3c48e545821